src/app/pages/login-page/login-form/login-form.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-login-form',
+  templateUrl: './login-form.component.html',
+  styleUrls: ['./login-form.component.scss'],
+})
+export class LoginFormComponent implements OnInit {
+  constructor() {}
+
+  LoginForm = new FormGroup({
+    username: new FormControl(),
+    password: new FormControl(),
+  });
+
+  ngOnInit(): void {}
+
+  onSubmit(): void {}
```

Comment:
Don't leave empty methods. if anything should be here in further PR's just add todo's (//TODO: implement submit functionality when the endpoint is ready)